packages/terra-form-select/src/_Menu.jsx

```diff
@@ -18,10 +20,18 @@ const propTypes = {
    * The content of the menu.
    */
   children: PropTypes.node,
+  /**
+   * Translation object to use for interpreting strings
+   */
+  intl: PropTypes.object,
   /**
    * Content to display when no results are found.
    */
   noResultContent: PropTypes.node,
+  /**
+   * String to send to screen readers to read out no results found.
+   */
+  noResultScreenReaderText: PropTypes.string,
```

Comment:
Is this used anywhere?

Reply:
It doesn't look like it is.

Reply:
https://github.com/cerner/terra-core/pull/2163/commits/be2f020b6c4484501b6d68bbde493987529c2899